WordPress/build.gradle

```diff
@@ -72,9 +72,11 @@ sentry {
         logcat.enabled = false
     }
     autoInstallation.enabled = false
-    includeSourceContext = true
-    autoUploadSourceContext = true
     includeDependenciesReport = false
+
+    Boolean isCi = System.getenv('CI')?.toBoolean() ?: false
+    includeSourceContext = isCi
+    includeProguardMapping = isCi
```

Comment:
Suggestion (💡): I am seeing various ways of doing the same with this `CI` related configuration in this project, and that made me wonder what's best, or more readable:

1. `System.getenv('CI')?.toBoolean() ?: false` versus
2. `System.getenv('CI') ? true : false` (PS: I like this more!)

Then, I wonder if having something like that is (actually) better than exposing a variable:

```groovy
    includeSourceContext = System.getenv('CI') ? true : false
    includeProguardMapping = System.getenv('CI') ? true : false
```

Then, me continuing thinking, we can actually do something more interesting, have `isCi` defined in the root level `build.gradle` related `ext { ... }` block, just like you did with `secretProperties` on this other PR, and reuse this everywhere as `rootProject.isCi`, just to avoid repetition, wdyt? 🤔

Reply:
The problem with this is that `CI=false ./gradlew assembleJetpackVanillaRelease` will send mapping files, and this is not something I'd expect when working with the project. 

> Then, me continuing thinking, we can actually do something more interesting, have isCi defined in the root level build.gradle related ext { ... } block, just like you did with secretProperties on this other PR, and reuse this everywhere as rootProject.isCi

Sure, why not! I'll see if this is possible - my only concern is Gradle lifecycle, e.g. if we'll be able to use this in settings plugins

Reply:
> The problem with this is that CI=false ./gradlew assembleJetpackVanillaRelease will send mapping files, and this is not something I'd expect when working with the project.

Oh, you're right @wzieba , true, hmmm...

`./gradlew assembleJetpackJalapenoDebug`
```
System.getenv('CI')?.toBoolean() ?: false -> false
System.getenv('CI') ? true : false -> false
System.getenv('CI') -> null
```

`CI=true ./gradlew assembleJetpackJalapenoDebug`
```
System.getenv('CI')?.toBoolean() ?: false -> true
System.getenv('CI') ? true : false -> true
System.getenv('CI') -> true
```

`CI=false ./gradlew assembleJetpackJalapenoDebug`
```
System.getenv('CI')?.toBoolean() ?: false -> false
System.getenv('CI') ? true : false -> true
System.getenv('CI') -> false
```

Thus, using `System.getenv('CI')?.toBoolean() ?: false` everywhere is much better/safer and more deterministic, in the sense that we (at least I) don't make the wrong assumptions again... 😅 Sure, we will never have `CI=false` but it is possible nevertheless and we should guard/account for that no matter.

---

> Sure, why not! I'll see if this is possible - my only concern is Gradle lifecycle, e.g. if we'll be able to use this in settings plugins

Thanks! 🙏

Reply:
Addressed in 79fe4cbc8a02080c9c1b1096a63237a2eec1da1c - unfortunatelly, I had to use global extension on `gradle` object, I think it still is an improvement, WDYT?

Reply:
I like it @wzieba , much better, and I don't mind too much about it being a global extension either, many thanks! 🙇 ❤️ 🚀